content/help/bugs/index.md

```diff
@@ -9,8 +9,48 @@ aliases:
 
 Oh no! You found a bug?
 
-Search our [issue database](https://github.com/istio/istio/issues/) to see if we already know about
-your problem and learn about when
-we think we can fix it. If you don't find your problem in the database, please open a [new issue](https://github.com/istio/istio/issues/new/choose)
-and let us know what's
-going on.
+Search our [issue database](https://github.com/istio/istio/issues/) to see if
+we already know about your problem and learn about when we think we can fix
+it. If you don't find your problem in the database, please open a [new
+issue](https://github.com/istio/istio/issues/new/choose) and let us know
+what's going on. Be sure to include a [cluster state
```

Comment:
I suggest creating a separate paragraph for the last sentence.

I suggest starting this paragraph with:

If you're running on Kubernetes, consider including a...